eclair-core/src/main/scala/fr/acinq/eclair/db/Databases.scala

```diff
@@ -35,24 +37,39 @@ trait Databases {
 
   val pendingRelay: PendingRelayDb
 
-  def backup(file: File) : Unit
+  def backup(file: File): Unit
 }
 
-object Databases {
+object Databases extends Logging {
```

Comment:
It seems like we should try to avoid extending this `Logging` trait and instead implicitly pass a logger to the functions that need one. Can you try doing that?

Reply:
Using `grizzled.slf4j.Logging` is indeed discouraged, but for code completely unrelated to actors I think it makes sense to keep using it. What we don't want is lose the context when we call code in static methods from within an actor.